packages/hint-compat-api/src/utils/alternatives.ts

```diff
@@ -0,0 +1,76 @@
+import { AlternativeDetails, UnsupportedBrowsers } from '@hint/utils/dist/src/compat/browsers';
+
+import { getMessage } from '../i18n.import';
+
+import { formatSupported } from './browsers';
+
+export type BrowserAlternativeDetails = AlternativeDetails & {
+    browser: string;
+};
+
+/**
+ * Sort alternatives into groups based on those recommending the same
+ * alternate feature.
```

Comment:
Nit: `alternative` 😛

Reply:
Some habits are hard to break 😉

Fixed.